configmodel/__init__.py

```diff
@@ -146,6 +196,15 @@ def path_to_list(path):
     return list(path)
 
 
+def path_to_env(path):
```

Comment:
i saw your comment about merging issue. just to confirm, do we want to keep this for future use?

Reply:
😅 whoops, I forgot to remove this function. Nice catch. Let's remove it, at least unless we find a clean way to make it happen.
I will remove it